.github/assets/winres.json

```diff
@@ -0,0 +1,78 @@
+{
+  "RT_GROUP_ICON": {
+    "APP": {
+      "0409": ".github/assets/terragrunt.png"
+    }
+  },
+  "RT_MANIFEST": {
+    "#1": {
+      "0409": {
+        "assembly": {
+          "identity": {
+            "name": "Terragrunt",
+            "version": "0.0.0.0"
+          },
+          "description": "Terragrunt - Orchestrate OpenTofu and Terraform at Scale"
+        },
+        "compatibility": {
+          "application": [
+            {
+              "supportedOS": {
+                "Id": "{e2011457-1546-43c5-a5fe-008deee3d3f0}",
+                "comment": "Windows Vista / Windows Server 2008"
+              }
+            },
+            {
+              "supportedOS": {
+                "Id": "{35138b9a-5d96-4fbd-8e2d-a2440225f93a}",
+                "comment": "Windows 7 / Windows Server 2008 R2"
+              }
+            },
+            {
+              "supportedOS": {
+                "Id": "{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}",
+                "comment": "Windows 8 / Windows Server 2012"
+              }
+            },
+            {
+              "supportedOS": {
+                "Id": "{1f676c76-80e1-4239-95bb-83d0f6d0da78}",
+                "comment": "Windows 8.1 / Windows Server 2012 R2"
+              }
+            },
+            {
+              "supportedOS": {
+                "Id": "{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}",
+                "comment": "Windows 10, Windows 11 / Windows Server 2016, 2019, 2022"
+              }
+            }
+          ]
+        },
+        "dpiAwareness": "PerMonitorV2, PerMonitor"
+      }
+    }
+  },
+  "RT_VERSION": {
+    "#1": {
+      "0409": {
+        "fixed": {
+          "file_version": "0.0.0.0",
+          "product_version": "0.0.0.0"
+        },
+        "info": {
+          "0409": {
+            "Comments": "Standardize IaC and manage growing infra complexity: define units, stacks, cut repetition with includes/hooks, execute modules in dependency order across environments",
+            "CompanyName": "Gruntwork, Inc.",
+            "FileDescription": "Terragrunt - Orchestrate OpenTofu and Terraform at Scale",
+            "FileVersion": "0.0.0",
+            "InternalName": "terragrunt",
+            "LegalCopyright": "Copyright (C) 2025 Gruntwork, Inc.",
+            "OriginalFilename": "terragrunt.exe",
+            "ProductName": "Terragrunt",
+            "ProductVersion": "0.0.0"
```

Comment:
NIT: this section makes me wonder if we should generate this json at build / sign time rather than have it checked in. Not necessary for the first version.